### PR TITLE
GL-548 - Update CreateVat code to handle samples that do not contain all population groups.

### PR DESCRIPTION
**Pull request by gbggrant**
Update GvsCreateVAT.wdl to build the subpopulation-specific files from the input ancestry file.
Have GvsCreateVAT.wdl only pull fields from the VCF for the selected subpopulations.
Update create_variant_annotation_table.py to set empty population-specific AC/AN/AF, etc. values if population is not present.

### CODE REVIEW

**Comment by gbggrant** [comment]
CreateVAT workflow run on a quickstart with only two populations for the ten samples (amr, afr) [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Quickstart%20v3%20ggrant/job_history/30527ce2-3705-4927-a1dc-512468a94a02).

**Review by rsasch — CHANGES_REQUESTED** [review]
Just one request: please update all the WDLs that use our custom docker image to use the most up-to-date one, "us.gcr.io/broad-dsde-methods/variantstore:gg_vattest_2022_07_28"

**Comment by gbggrant** [comment]
Sounds good - was thinking of doing that at the last moment (i.e. having the PR approved, build a 'ah_varstore_<date>' docker off of this PR and then update the wdls and then merge.

**Review by rsasch — APPROVED** [review]

File: scripts/variantstore/wdl/GvsCreateVAT.wdl [hunk 1]
```diff
@@ -23,7 +23,6 @@ workflow GvsCreateVAT {
     Array[String] contig_array = ["chr1", "chr2", "chr3", "chr4", "chr5", "chr6", "chr7", "chr8", "chr9", "chr10", "chr11", "chr12", "chr13", "chr14", "chr15", "chr16", "chr17", "chr18", "chr19", "chr20", "chr21", "chr22", "chrX", "chrY", "chrM"]
     File reference = "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.fasta"
     File nirvana_data_directory = "gs://broad-dsp-spec-ops/scratch/rcremer/Nirvana/NirvanaData.tar.gz"
-    File AnAcAf_annotations_template = "gs://broad-dsp-spec-ops/scratch/rcremer/Nirvana/vat/custom_annotations_template.tsv"
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsCreateVAT.wdl:26]: nice!

**Review by RoriCremer — APPROVED** [review]

**Pull request merged**